daemon/logger/fluentd/fluentd.go

```diff
@@ -24,12 +26,24 @@ type fluentd struct {
 }
 
 const (
-	name                             = "fluentd"
-	defaultHostName                  = "localhost"
-	defaultPort                      = 24224
-	defaultTagPrefix                 = "docker"
-	defaultIgnoreConnectErrorOnStart = false           // So that we do not break existing behaviour
-	defaultBufferLimit               = 1 * 1024 * 1024 // 1M buffer by default
+	name = "fluentd"
+
+	defaultHost        = "127.0.0.1"
+	defaultPort        = 24224
+	defaultBufferLimit = 8 * 1024 * 1024
```

Comment:
I'd personally prefer to keep this @ a 1MB buffer.
Consider you have 100 containers (which is not much) each with an 8MB buffer